ajv-keywords.d.ts

```diff
@@ -0,0 +1,28 @@
+declare module 'ajv-keywords' {
+  import { Ajv } from 'ajv';
+  
+  type AdditionalKeywords =
+    | 'typeof'
+    | 'instanceof'
+    | 'range'
+    | 'regexp'
+    | 'formatMinimum'
+    | 'formatMaximum'
+    | 'transform'
+    | 'uniqueItemProperties'
+    | 'allRequired'
+    | 'anyRequired'
+    | 'oneRequired'
+    | 'patternRequired'
+    | 'prohibited'
+    | 'deepProperties'
+    | 'deepRequired'
+    | 'switch'
+    | 'select'
+    | 'dynamicDefaults'
+    ;
+
+  function keywords(ajv: Ajv, include?: AdditionalKeywords | AdditionalKeywords[]): Ajv;
```

Comment:
I think it should be just `string | string[]`, Iet's please not make it the list of values.

Reply:
I have just pushed the requested change